### DISCUSSION TITLE
Managing DataValidation plugins in Avalonia 11

### DISCUSSION THREAD

**Discussion by 17steen** [AvaloniaUI/Avalonia#10910] [Q&A] [answered]
The docs mention how to deal with validation plugin clashes with a snippet: [Data Validation](https://docs.avaloniaui.net/docs/data-binding/data-validation#manage-validationplugins).
But the ExpressionObserver object shown in the snippet no longer seems to exist in Avalonia 11, I'm wondering what the alternative is.
I'm sorry if this is answered somewhere else, I tried finding the answer on my own but failed to do so, thanks in advance.

**Comment by timunie** [comment, marked as answer]
Very hidden it was answered here: https://github.com/AvaloniaUI/Avalonia/discussions/10480#discussioncomment-5111355

```cs
BindingPlugins.DataValidators.RemoveAt(0);
```

**Reply by 17steen** [reply]
thanks a lot for the quick answer!
